### PR TITLE
🎨 create the home landing page #2

### LINKED ISSUE #2: 🎨  Create the home landing page
#  Create the Home Landing page 
---
NB: This issue is dependent on #1

---
Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice hero section on the home page that captures my attention and introduces the page content. I want a good experience on the web and phone. Use the header created in #1.

### Description

Design:
HomeLandingPage

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528
```

### Assets.
Logo URL
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1695970671/elewa-education-website/Shared/elewa-logo_eocvkz.png
```

Right Arrow URL
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1695972080/elewa-education-website/Shared/toRightArrow_tlcdnk.svg
```
### Technical Analysis

This feature can be found in `index.html`.

### Acceptance Criteria

1. When rendered the hero is pixel-perfect compared to the design above.
4. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Schola-droid**
# Description

A navigation bar with links and a contact button.
Content includes a heading, a paragraph, and a button with an arrow icon.
An image with the class "feature-image
Responsiveness in desktop, tablet, and mobile devices.

Fixes # (#40 ) 

## Screenshot (optional)
Desktop:
Screenshot 2023-10-09 at 00 42 53

Tablet:
Screenshot 2023-10-09 at 00 48 51

Mobile:
Screenshot 2023-10-09 at 00 49 51

Did it with https://github.com/paul-nadola

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -1,5 +1,113 @@
...
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 100px;
+    height: 50px;
+    position: relative;
+    margin-top: 0px;
+    margin-left: 200px
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use single margin property here  
If the margin property has four values:

margin: 25px 50px 75px 100px;
top margin is 25px
right margin is 50px
bottom margin is 75px
left margin is 100px

File: src/ts/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+const b = "elewa"
+console.log(b);
```
Comment (JAPHETHNYARANGA) [inline src/ts/index.ts]: this is not necessary for the pull request

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Is the page responsive, i cannot see specific screen sizes specified.  https://www.w3schools.com/css/css_rwd_mediaqueries.asp

**Comment by Schola-droid** [comment]
> Is the page responsive, i cannot see specific screen sizes specified. https://www.w3schools.com/css/css_rwd_mediaqueries.asp

Working on it sir

File: src/css/style.css (hunk 1 above)
Comment (Schola-droid) [reply src/css/style.css]: > use single margin property here If the margin property has four values:
> 
> margin: 25px 50px 75px 100px; top margin is 25px right margin is 50px bottom margin is 75px left margin is 100px

Okay

File: src/ts/index.ts (hunk 2 above)
Comment (Schola-droid) [reply src/ts/index.ts]: > this is not necessary for the pull request

Noted sir.

File: src/css/style.css (hunk 1 above)
Comment (Schola-droid) [reply src/css/style.css]: Done with the revision @JAPHETHNYARANGA

**Review by JAPHETHNYARANGA — DISMISSED** [review]

**Review by JAPHETHNYARANGA — APPROVED** [review]

**Pull request merged**